Dockerfile

```diff
@@ -15,7 +15,7 @@ FROM node:14-alpine
 RUN npm install -g pm2
 WORKDIR /app
 COPY migrations migrations
-COPY ecosystem.config.js package.json .
+COPY ecosystem.config.js package.json ./
```

Comment:
How is `.` outdated? There's no harm using `./`, but I believe using just `.` is common, e.g. there are various places in https://docs.docker.com/develop/develop-images/dockerfile_best-practices/ uses where just `.` is used to refer to the current working directory.